docs/tutorials/tutorial-delete-data.md

```diff
@@ -103,29 +103,17 @@ In the [segments view](http://localhost:8888/unified-console.html#segments), cli
 
 The top of the info box shows the full segment ID, e.g. `deletion-tutorial_2015-09-12T14:00:00.000Z_2015-09-12T15:00:00.000Z_2019-02-28T01:11:51.606Z` for the segment of hour 14.
 
-Let's disable the hour 13 and 14 segments by sending a POST request to the Coordinator with this payload
-
-```json
-{
-  "segmentIds":
-  [
-    "deletion-tutorial_2015-09-12T13:00:00.000Z_2015-09-12T14:00:00.000Z_2019-05-01T17:38:46.961Z",
-    "deletion-tutorial_2015-09-12T14:00:00.000Z_2015-09-12T15:00:00.000Z_2019-05-01T17:38:46.961Z"
-  ]
-}
-```
-
-This payload json has been provided at `quickstart/tutorial/deletion-disable-segments.json`. Submit the POST request to Coordinator like this:
-
+Let's disable the last two segments, hour 22 and 23 segments, by sending a POST request to the Coordinator with this payload:
 ```bash
-curl -X 'POST' -H 'Content-Type:application/json' -d @quickstart/tutorial/deletion-disable-segments.json http://localhost:8081/druid/coordinator/v1/datasources/deletion-tutorial/markUnused
+unusedSegmentIds=$(curl -X 'GET' -H 'Content-Type:application/json' http://localhost:8081/druid/coordinator/v1/datasources/deletion-tutorial/segments | jq '.[-2:]')
```

Comment:
Since these are two separate commands, we should include them separately. Could introduce it with something like the following:
>Disable the last two segments, hour 22 and 23 segments, by sending a POST request to the Coordinator with the corresponding segment IDs.
>The following command queries the Coordinator for segment IDs and uses `jq` to parse and extract the IDs of the last two segments. The segment IDs are stored in an environment variable named `unusedSegmentIds`.
> [code block with cmd]
>
>The following POST request marks the segments unused:

Reply:
We should also list at the top of the doc that jq is a prerequisite